### PR TITLE
Refactor transaction and submission creation 

### PR DESCRIPTION
**Pull request by AlisonGriffiths**
- Moved new submission route after psc list
- Fixed tests
- Removed redundant psc list post method
- Updated psc verify link for create new submission

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/handlers/individual-psc-list/individualPscListHandler.ts [hunk 1]
```diff
@@ -58,32 +55,28 @@ export class IndividualPscListHandler extends GenericHandler<IndividualPscListVi
...
 
         function resolveUrlTemplate (prefixedUrl: string): string | null {
-            return addSearchParams(getUrlWithTransactionIdAndSubmissionId(prefixedUrl, req.params.transactionId, req.params.submissionId), { lang, pscType });
+            return addSearchParams(prefixedUrl, { companyNumber, lang });
         }
     }
 
-    public async executeGet (req: Request, res: Response): Promise<ViewModel<IndividualPscListViewData>> {
-        logger.info(`${IndividualPscListHandler.name} - ${this.executeGet.name} called for transaction: ${req.params?.transactionId} and submissionId: ${req.params?.submissionId}`);
+    public async execute (req: Request, res: Response): Promise<ViewModel<IndividualPscListViewData>> {
```
Comment (eedwards0) [inline src/routers/handlers/individual-psc-list/individualPscListHandler.ts:78]: Semantics really - but would it be clearer to keep as `executeGet`?

File: src/routers/individualPscListRouter.ts [hunk 2]
```diff
@@ -4,15 +4,10 @@ import { IndividualPscListHandler } from "./handlers/individual-psc-list/individ
 
 const individualPscListRouter: Router = Router({ mergeParams: true });
 
-individualPscListRouter.get("/", handleExceptions(async (req: Request, res: Response, _next: NextFunction) => {
+individualPscListRouter.all("/", handleExceptions(async (req: Request, res: Response, _next: NextFunction) => {
     const handler = new IndividualPscListHandler();
-    const { templatePath, viewData } = await handler.executeGet(req, res);
+    const { templatePath, viewData } = await handler.execute(req, res);
```
Comment (eedwards0) [inline src/routers/individualPscListRouter.ts:9]: Keep as `executeGet`?

**Review by eedwards0 — APPROVED** [review]
All working as expected - just query about semantics - on prefixing the `execute` functions with 'get' for the PSC list functions and maybe  'post' for the new submission.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by eedwards0 — APPROVED** [review]

**Pull request merged**